notebooks/getting_started/README.md

```diff
@@ -0,0 +1,50 @@
+# VRS Python Getting Started Notebook Series
+This series of notebooks is intended to be the fastest way to becoming productive with the 
+[vrs-python](https://github.com/ga4gh/vrs-python) package. The 
+intent of each notebook in this series is to be an interactive introduction to functionally contained in the vrs-python package.
+
+A beginning developer level of familiarity with python, jupyter notebooks is assumed in order to run this notebook series. 
+You should be familiar with installing packages and running commands in your execution environment.
+
+### Pre-requisites
+The following software packages must exist in your execution environment before running these notebooks:
+* git
+* postgresql@14
+* python@3.10
+* make
+
+### Setup vrs-python
+From a terminal window, run the following commands:
+* git clone --recurse-submodules https://github.com/ga4gh/vrs-python
+* cd vrs-python
+* make devready
+* source venv/3.10/bin/activate
+* cd notebooks/getting_started 
+* jupyter notebook "the notebook name.ipynb"
+
+## Notebooks
```

Comment:
Made a comment about removing spaces in the names, will need to update the reference links below and then they will work they currently don't because of the spaces. Would have needed to do `[Quick Start](1%20Quick%20Start.ipynb)`